aion/test/controllers/room_controller_test.exs

```diff
@@ -1,10 +1,11 @@
 defmodule Aion.RoomControllerTest do
   use Aion.AuthConnCase
 
-  alias Aion.{Room, Question, User}
+  alias Aion.{Room, Question, User, Category, RoomCategory}
   alias Aion.RoomChannel.{Monitor, QuestionSet}
 
   @room %Room{description: "Here come dat boi", name: "Dat Boi"}
+  @invalid_room %Room{name: "This is an invalid data, has no description field"}
```

Comment:
`The provided data is invalid, it does not contain the description field`
or 
`The provided data is invalid does not contain the description field`.

The data is singular uncountable, so there should be no `an` before it